src/command-reader.ts

```diff
@@ -20,14 +20,14 @@ class CommandReader {
         // Execute command
         const { command, args, rawArgs } = parsedCommand;
         if (Object.prototype.hasOwnProperty.call(commands, command)) {
-            const commandInfo: Command = commands[command];
+            const commandInfo: Command = commands[command as CommandName];
             if (commandInfo.privileged && !state.isAdmin(msg.author.id)) {
                 msg.channel.send('You can\'t do that');
             } else if (commandInfo.failIfDisabled && state.isDisabled()) {
                 msg.channel.send('I can\'t do that while I\'m disabled');
             } else {
                 try {
-                    commandInfo.fn(msg, rawArgs, ...args);
+                    commandInfo.fn && commandInfo.fn(msg, rawArgs, ...args);
```

Comment:
Similar to above comment, can you make this a proper if-statement and reply with a warning in the else case?